src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -2674,101 +2703,103 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
-                            rNumericArgs(2) * state.dataHeatBal->Zone(state.dataHeatBal->Mixing(Loop).ZonePtr).FloorArea;
-                        if (state.dataHeatBal->Zone(state.dataHeatBal->Mixing(Loop).ZonePtr).FloorArea <= 0.0) {
-                            ShowWarningError(state,
-                                             std::string{RoutineName} + cCurrentModuleObject + "=\"" + cAlphaArgs(1) + "\", " + cAlphaFieldNames(4) +
-                                                 " specifies " + cNumericFieldNames(2) + ", but Zone Floor Area = 0.  0 Mixing will result.");
-                        }
-                    } else {
-                        ShowSevereError(state,
-                                        format("{}{}=\"{}\", invalid flow/person specification [<0.0]={:.3R}",
-                                               RoutineName,
-                                               cCurrentModuleObject,
-                                               cAlphaArgs(1),
-                                               rNumericArgs(2)));
-                        ErrorsFound = true;
+        AirflowSpec flow = static_cast<AirflowSpec>(getEnumerationValue(airflowNamesUC, cAlphaArgs(4))); // NOLINT(modernize-use-auto)
+        switch (flow) {
+        case AirflowSpec::Flow:
+        case AirflowSpec::FlowPerZone:
+            state.dataHeatBal->Mixing(Loop).DesignLevel = rNumericArgs(1);
+            if (lAlphaFieldBlanks(1)) {
+                ShowWarningError(state,
+                                 std::string{RoutineName} + cCurrentModuleObject + "=\"" + cAlphaArgs(1) + "\", " + cAlphaFieldNames(4) +
+                                     " specifies " + cNumericFieldNames(1) + ", but that field is blank.  0 Mixing will result.");
+            }
+            break;
+
+        case AirflowSpec::FlowPerArea:
+            if (state.dataHeatBal->Mixing(Loop).ZonePtr != 0) {
+                if (rNumericArgs(2) >= 0.0) {
+                    state.dataHeatBal->Mixing(Loop).DesignLevel =
+                        rNumericArgs(2) * state.dataHeatBal->Zone(state.dataHeatBal->Mixing(Loop).ZonePtr).FloorArea;
+                    if (state.dataHeatBal->Zone(state.dataHeatBal->Mixing(Loop).ZonePtr).FloorArea <= 0.0) {
+                        ShowWarningError(state,
+                                         std::string{RoutineName} + cCurrentModuleObject + "=\"" + cAlphaArgs(1) + "\", " + cAlphaFieldNames(4) +
+                                             " specifies " + cNumericFieldNames(2) + ", but Zone Floor Area = 0.  0 Mixing will result.");
                     }
+                } else {
+                    ShowSevereError(state,
+                                    format("{}{}=\"{}\", invalid flow/person specification [<0.0]={:.3R}",
+                                           RoutineName,
+                                           cCurrentModuleObject,
+                                           cAlphaArgs(1),
```

Comment:
Use `thisVentilation.Name` instead of `cAlphaArgs(1)` if already captured.

Reply:
Incidentally, why is `RoutineName` even needed here? Users don't know routine names and don't all of these errors come from `CMO::GetInput` anyway?